sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -273,3 +228,75 @@ export class RequestResponseLink implements ReqResLink {
     return new RequestResponseLink(session, sender, receiver);
   }
 }
+
+type NormalizedInfo = {
+  statusCode: number;
+  statusDescription: string;
+  errorCondition: string;
+};
+
+// Handle different variations of property names in responses emitted by EventHubs and ServiceBus.
+const getCodeDescriptionAndError = (props: any): NormalizedInfo => {
```

Comment:
It'd be good to add some tests for this so we can see what the EventHubs event and the Service Bus message look like that you're converting.

Reply:
Added tests